rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_reader :person, :book
```

Comment:
Both `person` and `book` instance variable should have a `getter` and a `setter`. However, in your case, you have defined `attr_reader` for them which only sets a `getter`. Kindly use `attr_accessor` to set both `getter` and `setter`.